examples/prove_and_verify/prove_and_verify.sh

```diff
@@ -3,12 +3,12 @@ set -eu
 
 BACKEND=${BACKEND:-bb}
 
-nargo execute witness
+cargo run execute --pedantic-solving witness
```

Comment:
Why did this change from nargo to cargo?

Reply:
Leftover from testing: reverted